test/test.js

```diff
@@ -348,15 +368,9 @@ describe( 'rollup', function () {
 				( config.skip ? it.skip : config.solo ? it.only : it )( dir, function ( done ) {
 					process.chdir( path.resolve( CLI, dir ) );
 
-					if (os.platform() === 'win32') {
-						config.command = "node " + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
-					}
+					const command = 'node ' + path.resolve( __dirname, '../bin' ) + path.sep + config.command;
```

Comment:
@TrySound confused brah. I see the words you are writing but can't put together the point.

Again... net/net is the same. What am I missing here?
![capture d ecran 2016-09-06 a 03 24 24](https://cloud.githubusercontent.com/assets/38223/18264983/57f41cfa-73e1-11e6-8436-43c353ad09e5.png)